sidekick_core/lib/src/repository.dart

```diff
@@ -1,16 +1,13 @@
-import 'dart:io';
-
 import 'package:sidekick_core/sidekick_core.dart';
-import 'package:sidekick_core/src/sidekick_context.dart';
+import 'package:sidekick_core/src/dart_package.dart' as dartpackage;
 
 /// Finds the root of the repo
-@Deprecated('Use SidekickContext') // TODO be more precise
-Repository findRepository() => Repository(root: SidekickContext.repository);
+@Deprecated('Use SidekickContext.repository') // TODO be more precise
+Repository findRepository() => SidekickContext.repository;
 
 /// The Git repository of the project
 ///
 /// Might contain a single dart project or multiple packages, or even non dart packages
-@Deprecated('Use SidekickContext') // TODO be more precise
```

Comment:
I don't think we need `Repository` anymore, if so let's add the deprecation again